lib/squadster_web/resolvers/formations.ex

```diff
@@ -60,6 +60,13 @@ defmodule SquadsterWeb.Resolvers.Formations do
     {:error, "Access denied"}
   end
 
+  def update_squad_members(_parent, %{batch: args}, %{context: %{current_user: user}}) do
+    case Formations.bulk_update_squad_members(args,user) do
+      {:ok, result} -> {:ok, Enum.map(result, fn {index,element} -> element end)}
+      nil -> {:error, "Not enough permissions"}
```

Comment:
Will it return nil when Permissions will deny access (just to be sure)? Also, not a big deal, but spaces in args on lines 64 and 65 is missing

Reply:
I've got `nil` while tested mutation, that's why it here

Reply:
Ok, that's nice